ooniprobe/AppDelegate.m

```diff
@@ -15,12 +15,13 @@ @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    SharkORM.settings.sqliteJournalingMode = @"DELETE";
+
     if ([self isUITestingEnabled])
         [self copyDBTesting];
         
     [SharkORM setDelegate:self];
     //Default is WAL https://www.sqlite.org/wal.html
```

Comment:
This comment used to apply to the code that you moved; can you also move this comment?